metadata_mapper/mappers/mapper.py

```diff
@@ -120,13 +120,11 @@ def map_is_shown_by(self) -> Union[str, None]:
 
     # Mapper Helpers
     def collate_subfield(self, field: str, subfield: str) -> list:
-        return [f[subfield] for f in self.source_metadata.get(field, [])]
+        return [f[subfield] for f in self.source_metadata.get(self, field, [])]
```

Comment:
`get` takes 2 args. I assume the `self` is leftover from a different change?

Reply:
Derps, fixing.